snapshots/biodiversity/2023-01-11/cherry_blossom.csv.dvc

```diff
@@ -1,18 +1,32 @@
 meta:
-  name: Peak Cherry Tree Blossom at Kyoto City (2022)
-  source_name: Multiple datasets compiled by Aono (2021)
-  publication_year: 2021
-  publication_date: 2021-03-26
-  url: http://atmenv.envi.osakafu-u.ac.jp/aono/kyophenotemp4/
-  source_data_url:
-  license_url: http://atmenv.envi.osakafu-u.ac.jp/aono/kyophenotemp4/
-  license_name:
-  date_accessed: 2023-01-11
-  is_public: true
-  description: |
+  origin:
+    producer: Yasuyuki Aono
+    dataset_title_owid: Peak Cherry Tree Blossom at Kyoto City
+    dataset_url_main: http://atmenv.envi.osakafu-u.ac.jp/aono/kyophenotemp4/
+    attribution: Multiple sources compiled by Aono
+    attribution_short: Aono
```

Comment:
Is this definition necessary? I suppose `attribution_short` replaces `producer` and in this case I think "Yasuyuki Aono" is probably short enough, and we won't need to just say "Aono" anywhere.

Reply:
Yeah let's go for the full name here